src/main/java/de/qabel/desktop/ui/about/AboutController.java

```diff
@@ -117,15 +152,47 @@ private void indexJarNames() throws ParserConfigurationException, IOException, S
         }
     }
 
+    private void setDesktopVersion() {
+        appVersion.setText(appVersion.getText() + " " + currentVersion);
+
+    }
+
     private void activateButtons() {
-        if (thanksFileContent.isEmpty()) {
-           thanksButton.setDisable(true);
+        if (aboutFilesContent.QAPLContent.isEmpty()) {
+            QAPLBox.setDisable(true);
+        }
+        if (aboutFilesContent.imprintContent.isEmpty()) {
+            imprintBox.setDisable(true);
+        }
+        if (aboutFilesContent.silLicenseContent.isEmpty()) {
+            silLicenseBox.setDisable(true);
+            sourceSansBox.setDisable(true);
+        }
+        if (aboutFilesContent.thanksFileContent.isEmpty()) {
+            thanksBox.setDisable(true);
+        }
+        if (aboutFilesContent.attributionLicenseContent.isEmpty()) {
+            attributionLicenseBox.setDisable(true);
+        }
+        if (aboutFilesContent.creativeLicenseContent.isEmpty()) {
+            creativeLicenseBox.setDisable(true);
+        }
+        if (aboutFilesContent.lgplLicenseContent.isEmpty()) {
+            lgplLicenseBox.setDisable(true);
+        }
+        if(aboutFilesContent.privateNotesContent.isEmpty()) {
```

Comment:
please make these statements fit the code style:
`if (...) {`
(one space in front of the round brackets and one space behind)